_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -87912,10 +87912,8 @@
 /area/crew_quarters/abandoned_gambling_den)
 "dob" = (
 /obj/structure/table/wood/poker,
-/obj/item/stack/spacecash/c1000{
```

Comment:
... fair enough.

could you maybe replace this with a stack of a random amount of money, though? like, can't you just use a coin/space cash loot table or something?

Reply:
don't we have the 10 credit ones just for this reason

Reply:
exactly, the 10 credit ones can pad out the pool, with low chances for higher-denomination cash stacks or something.